src/tests/CountriesList.test.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import { MemoryRouter } from 'react-router-dom';
+import { Provider } from 'react-redux';
+import configureStore from 'redux-mock-store';
+import CountriesList from '../components/CountriesList'; // Double-check the path to your component
+
+// Create a mock Redux store
+const mockStore = configureStore([]);
+const store = mockStore({
+  pollutions: {
+    searchFilter: false,
+  },
+});
+
+// Describe your test
+describe('CountriesList component', () => {
+  // Test rendering the component
+  it('should render correctly', () => {
+    const component = renderer.create(
+      <Provider store={store}>
+        <MemoryRouter>
+          <CountriesList />
+        </MemoryRouter>
+      </Provider>,
+    );
+
+    const tree = component.toJSON();
+    expect(tree).toMatchSnapshot();
+  });
```

Comment:
- [ ] As per [this](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md#:~:text=Create%20unit%20tests,the%20Redux%20Store.) requirement,  kindly add more unit tests, integration tests, and tests that simulate user interactions.